flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/harness/TableAggregateHarnessTest.scala

```diff
@@ -56,7 +56,7 @@ class TableAggregateHarnessTest(mode: StateBackendMode) extends HarnessTestBase(
   @TestTemplate
   def testTableAggregate(): Unit = {
     val top3 = new Top3WithMapView
-    tEnv.registerFunction("top3", top3)
+    tEnv.createTemporarySystemFunction("top3", top3)
```

Comment:
here we can not use 
```scala
tEnv.createTemporarySystemFunction("top3", classOf[Top3WithMapView])
``` 
since `top3` is used in `flatAggregate` below

Reply:
fair enough! Thanks!